internal/payloads/reservation_payload.go

```diff
@@ -50,6 +59,12 @@ type AWSReservationResponsePayload struct {
 
 	// The ID of the aws reservation which was created.
 	AWSReservationID string `json:"aws_reservation_id"`
+
+	// Optional name of the instance(s).
+	Names *[]string `json:"names"`
+
+	// Immediately power off the system after initialization
+	PowerOff bool `json:"poweroff"`
 }
```

Comment:
I realized that the PowerOff field was not present in AWS payload by accident, so this fixes it.